lib/coinbase/address/wallet_address.rb

```diff
@@ -95,6 +95,18 @@ def trade(amount, from_asset_id, to_asset_id)
       trade
     end
 
+    # Signs the given unsigned payload.
+    # @param unsigned_payload [String] The hex-encoded hashed unsigned payload for the Address to sign.
+    # @return [Coinbase::PayloadSignature] The payload signature
+    def sign(unsigned_payload:)
+      ensure_can_sign!
+
+      signature = Coinbase.use_server_signer? ? nil : "0x#{@key.sign(Eth::Util.hex_to_bin(unsigned_payload))}"
```

Comment:
Can we use `Eth::Util.prefix_hex`?

Also can just do the following since ruby hoists the variable definition:
```suggestion
      unless Coinbase.use_server_signer?
        signature = Eth::Util.prefix_hex(@key.sign(Eth::Util.hex_to_bin(unsigned_payload)))
      end
```

Fine to leave as a ternary, but can be more difficult to read.

Reply:
updated to use `unless` clause

Reply:
And the truth is that I even lost my job because a lot of things happened. I don't know, I don't know if you can help me, I don't know how I got here and I try to do the best I can, I like to learn, but please help me, because the truth is that I'm practically out on the street, I have no money, I live in Atlanta, Georgia and as I repeat, I even lost my job because they cancelled my credit cards, how can I tell you, what a mess it made me, I don't know if you can help me.

Reply:
answer you something very humbly. I don't know what to do anymore, my dear friends, I have tried to speak a little English and I try to do it better than I can, but I don't know anymore, I am already, I don't even know how I got into this portal of wanting to get into my account and I just wanted to get my money back because I wanted to get into my account.

Reply:
and the truth is that since Sunday I have been in Combis for a long time. I had never had this setback, the truth is for me, that's why I liked the application because it has what I need and it's nice to go learning. But today I really don't know and I try, the truth is that I don't know. There I have gotten into quite a few things and I have tried to follow El Paso. Maybe I don't understand them, but I try to get the idea. Another thing is that I don't know much English. There is little that I can sometimes I don't answer because I am translating, it takes me a while.